src/hotspot/share/runtime/threads.cpp

```diff
@@ -552,6 +554,15 @@ jint Threads::create_vm(JavaVMInitArgs* args, bool* canTryAgain) {
   // crash Linux VM, see notes in os_linux.cpp.
   main_thread->stack_overflow_state()->create_stack_guard_pages();
 
+  if (UseSystemMemoryBarrier) {
+    if (!SystemMemoryBarrier::initialize()) {
+      // UseSystemMemoryBarrier = false;
+      vm_shutdown_during_initialization("Failed to initialize request system memory barrier synchronization.");
```

Comment:
typo: s/initialize request system/initialize the requested system/

Reply:
Fixed